src/scripts/main.js

```diff
@@ -1 +1,66 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+const promise1 = new Promise((resolve, reject) => {
+  body.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(reject, 3000);
```

Comment:
The `reject` function in `promise1` is called without any arguments. It's a good practice to pass an error message or object to `reject` to provide more context about the rejection.